csharp-advanced-topics/DeprecationInCsharp/DeprecationInCsharp/StringUtils.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Globalization;
+
+namespace Deprecation
+{
+    public class StringUtils
+    {
+        public const string input = "Hello, World!";
+        public const string reverseInput = "!dlroW ,olleH";
```

Comment:
These should be in the testing class, be private and I would suggest that the second string be renamed expectedResult, or expectedOutput